demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -53,6 +57,12 @@
     ERROR_MSG_TEMPLATE,
     IsEntityNameContainExcludedWordValidator,
 )
+from demisto_sdk.commands.validate.validators.BA_validators.BA113_is_content_item_name_contain_trailing_spaces import (
+    ContentTypes as ContentTypes113,
+)
+from demisto_sdk.commands.validate.validators.BA_validators.BA113_is_content_item_name_contain_trailing_spaces import (
```

Comment:
don't we have a ruff check for importing twice from the same file and not as `from x import y,z` ? 🤔